server/controllers/prisonerSummary/prisonerSummary.ts

```diff
@@ -0,0 +1,66 @@
+import type express from 'express'
+import type FormWizard from 'hmpo-form-wizard'
+
+import { BaseController } from '../index'
+import type { ReportWithDetails } from '../../data/incidentReportingApi'
+import { prisonerInvolvementOutcomes, prisonerInvolvementRoles } from '../../reportConfiguration/constants'
+
+export default class PrisonerSummary extends BaseController {
+  middlewareSetup() {
+    super.middlewareSetup()
+  }
+
+  locals(req: FormWizard.Request, res: express.Response) {
+    const locals = super.locals(req, res)
+    const reportId = res.locals.report.id
+    const report = res.locals.report as ReportWithDetails
+
+    const prisonerInvolvementLookup = Object.fromEntries(
+      prisonerInvolvementRoles.map(role => [role.code, role.description]),
+    )
+    const prisonerOutcomeLookup = Object.fromEntries(
+      prisonerInvolvementOutcomes.map(outcome => [outcome.code, outcome.description]),
+    )
+    let showTable = false
+    if (report.prisonersInvolved && report.prisonersInvolved.length > 0) {
+      showTable = true
+    }
+
+    let tableHeading = [{}]
+    if (report.createdInNomis) {
+      tableHeading = [
+        { text: 'Prisoner' },
+        { text: 'Role' },
+        { text: 'Outcome' },
+        { text: 'Details' },
+        { text: 'Action' },
+      ]
+    } else {
+      tableHeading = [{ text: 'Prisoner' }, { text: 'Role' }, { text: 'Details' }, { text: 'Action' }]
+    }
+
+    const backLink = `/reports/${reportId}`
```

Comment:
it might be a little better to use form-wizard machinery for the back link. you can add this method instead

```typescript
  getBackLink(_req: FormWizard.Request, res: express.Response): string {
    const reportId = res.locals.report.id
    return `/reports/${reportId}`
  }
```